docs/contribute/general/contribution_attribution.rst

```diff
@@ -0,0 +1,69 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+
+.. _contribution_attribution:
+
+Actions to Ensure Proper Contribution Attribution in Eclipse SCORE
+=====================================================================
+
+To ensure that contributions are correctly attributed to organizations in the **Eclipse SCORE** project, follow these steps:
+
+Actions for Contributors
+--------------------------
+
+1. **Create an Eclipse Foundation Account**
+   - If you don't have one, register at `Eclipse Foundation <https://accounts.eclipse.org/user/register>`_.
+
+2. **Link Your Account to Your Employer**
+   - Ensure that your **employer information** is set to a valid **Eclipse Foundation Member Company**.
+   - This can be done in the **Eclipse Foundation account settings**.
+
+3. **Use Your Corporate Email for Commits**
+   - Configure Git to use your corporate email associated with your Eclipse account::
+
+       git config --global user.email "your_corporate_email@yourcompany.com"
+
+   - Verify that the email is added and confirmed in **GitHub account settings**.
+
+4. **Avoid Using Different Credentials for Commits**
+   - If you've already committed using a different email, update your Git settings and **ensure future commits** match your Eclipse Foundation email.
+
+
+
+Actions for Employers
+----------------------
+
+1. **Ensure Employees Have an Eclipse Foundation Account**
+   - Encourage employees to **register and link their employer** to their account.
```

Comment:
+ When the employer account is ambiguous (subsidiaries etc), provide clear guidelines to employees which one shall be used.